src/actions/receive-profile.js

```diff
@@ -83,6 +83,9 @@ let requestIdleCallbackPolyfill: (
 
 if (typeof window === 'object' && window.requestIdleCallback) {
   requestIdleCallbackPolyfill = window.requestIdleCallback;
+} else if (typeof process === 'object' && process.nextTick) {
+  // Node environment
+  requestIdleCallbackPolyfill = process.nextTick;
```

Comment:
This is done so that we can easily use fake timers and still have requestIdleCallback works